core/src/main/scala/com/salesforce/op/stages/impl/classification/BinaryClassificationModelSelector.scala

```diff
@@ -60,7 +60,8 @@ case object BinaryClassificationModelSelector extends ModelSelectorFactory {
      * off by default
      */
     val modelTypesToUse: Seq[BinaryClassificationModelsToTry] = Seq(
-      MTT.OpLogisticRegression, MTT.OpRandomForestClassifier, MTT.OpGBTClassifier
+      MTT.OpLogisticRegression, MTT.OpRandomForestClassifier,
```

Comment:
I think it makes more sense to replace GBTs with xgboost, rather than having both.